ext/gd/libgd/gd_webp.c

```diff
@@ -57,7 +57,7 @@ gdImagePtr gdImageCreateFromWebpCtx (gdIOCtx * infile)
 			if (filedata) {
 				gdFree(filedata);
 			}
-			gd_error("WebP decode: realloc failed");
+			zend_error(E_ERROR, "WebP decode: realloc failed");
```

Comment:
I would rather override

https://github.com/php/php-src/blob/09d61b63688fb0fb58afcb0e105c9d0705ef735b/ext/gd/libgd/gd.c#L94

accordingly instead of having that area not using this function but zend error

or why is it needed here specifically?

Reply:
it is a follow-up of [this comment](https://github.com/php/php-src/commit/b456ae8d34bbdd6bb50c789ec92ded39a6fdb4f5#r144212449).